examples/golang/components/http-echo-tinygo/wasmcloud.toml

```diff
@@ -24,4 +24,4 @@ claims = [
   "wasmcloud:builtin:logging",
 ]
 wit_world = "echo"
-wasm_target = "wasm32-wasi"
+wasm_target = "wasm32-wasi-preview2"
```

Comment:
target should be wasm32-wasi-wasip2

Reply:
Actually realizing we haven't updated everywhere in wasmCloud to only refer to wasm32-wasip2 so OK if not fixed in this PR. Ref [rust wasi-targets](https://blog.rust-lang.org/2024/04/09/updates-to-rusts-wasi-targets.html).